apps/wallet-dashboard/components/transactions/TransactionsList.tsx

```diff
@@ -20,5 +21,16 @@ export function TransactionsList() {
         return <TransactionTile transaction={transaction} />;
     };
 
-    return <VirtualList items={transactions || []} estimateSize={() => 60} render={virtualItem} />;
+    return (
+        <VirtualList
+            items={allTransactions || []}
+            estimateSize={() => 60}
+            render={virtualItem}
+            fetchNextPage={fetchNextPage}
+            hasNextPage={hasNextPage}
+            isFetchingNextPage={isFetchingNextPage}
+            heightClassName="h-96"
+            overflowClassName="overflow-y-auto"
```

Comment:
this breaks the layout a bit, there is no other way to make the intersection observer work and keep the previous layout?

Reply:
we don't need the intersection observer, we just need to improve the css a bit